aerospike-mixin/dashboards/aerospike-overview.libsonnet

```diff
@@ -0,0 +1,1176 @@
+};
+
+local clientUDFTransactionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(aerospike_cluster, job) (rate(aerospike_namespace_client_udf_complete{aerospike_cluster=~"$aerospike_cluster", job=~"$job"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{aerospike_cluster}} - complete',
+    ),
+    prometheus.target(
+      'sum by(aerospike_cluster, job) (rate(aerospike_namespace_client_udf_error{job=~"$job", aerospike_cluster=~"$aerospike_cluster"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{aerospike_cluster}} - error',
+    ),
+    prometheus.target(
+      'sum by(aerospike_cluster, job) (rate(aerospike_namespace_client_udf_filtered_out{job=~"$job", aerospike_cluster=~"$aerospike_cluster"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{aerospike_cluster}} - filtered',
+    ),
+    prometheus.target(
+      'sum by(aerospike_cluster, job) (rate(aerospike_namespace_client_udf_timeout{job=~"$job", aerospike_cluster=~"$aerospike_cluster"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{aerospike_cluster}} - timeout',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Client UDF transactions',
+  description: 'Rate of client UDF transactions in an Aerospike cluster organized by result.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 20,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'normal',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'ops',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local connectionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(aerospike_cluster, job) (aerospike_node_stats_client_connections{aerospike_cluster=~"$aerospike_cluster", job=~"$job"})',
```

Comment:
so are these all gauge metrics and not counters? i notice you're not doing an `increase` or `rate` here

Reply:
Yep, [AS metric reference](https://docs.aerospike.com/reference/metrics) calls them out as instantaneous (aka gauges)